Core/Utilities/ElasticSearchDB.py

```diff
@@ -101,7 +109,9 @@ def getIndexes( self ):
     """
     It returns the available indexes...
     """
-    return [ index for index in self.__client.indices.get_aliases() ]
+    #we only return indexes which belong to a specific setup for example 'lhcb-production' or 'dirac-production etc.
+    #this is a policy used by IT to distinguish between users.
+    return [ index for index in self.__client.indices.get_aliases( "%s-*", self.__indexPrefix ) ]
```

Comment:
if `self.__indexPrefix == ''` then this may end up being an issue. Also, the `-` in `"%s-*", self.__indexPrefix` does not look necessarily standard to me.